Reference/Searching/Examine/index-v7.md

```diff
@@ -0,0 +1,37 @@
+---
+versionFrom: 7.0.0
+---
+
+# Examine
+
+_Examine uses Lucene as its search and index engine. Searching using Examine with Lucene can be very powerful and fast and it can also be really simple to setup and use._
+
+## What is Examine?
+
+Examine allows you to index and search data quickly and easily. It is a library that sits on top of Lucene, a very high performance search engine library written in Java. Examine is built on top of a .NET-implementation of Lucene to provide very efficient APIs to make searching and indexing as straight forward as possible. Umbraco provides a further layer on top, UmbracoExamine, that opens up the Umbraco-specific APIs for indexing and searching content and media out of the box.
```

Comment:
This block should be:

> Examine allows you to index and search data quickly and easily. Examine is a library that sits on top of Lucene.Net, a high performance search engine library. Examine provides easy to use APIs to make searching and indexing as straight forward as possible. Umbraco provides a further layer on top, UmbracoExamine, that opens up the Umbraco-specific APIs for indexing and searching content and media out of the box.

probably with a link to the lucene.net website: https://lucenenet.apache.org/

it's not written in Java! ... it's a c# port of the original Java source so best not to confuse people.